deployment/ccip/changeset/v1_5_1/cs_configure_token_pools.go

```diff
@@ -44,6 +46,7 @@ import (
 var (
 	_                           cldf.ChangeSet[ConfigureTokenPoolContractsConfig] = ConfigureTokenPoolContractsChangeset
 	ConfigureMultipleTokenPools                                                   = cldf.CreateChangeSet(ConfigureMultiplePoolLogic, configureMultiplePoolPreconditionValidation)
+	_                           cldf.ChangeSet[ConfigureTokenPoolContractsConfig] = UpdateGroupsTokenPool
```

Comment:
v2 changesets please, so we can separate the prevalidation from the execution logic